runtime/vm/createramclass.cpp

```diff
@@ -717,6 +736,7 @@ computeVTable(J9VMThread *vmStruct, J9Class *superclass, J9ROMClass *taggedClass
 	if (NULL == vTableAddress) {
 		if (maxSlots > vm->vTableScratchSize) {
 			vTableAddress = (UDATA *)j9mem_allocate_memory(maxSlots, J9MEM_CATEGORY_CLASSES);
+			vTableAllocated = true;
```

Comment:
We never write this back to the `vm->vTableScratch`?

Reply:
vTableScratch and vTableScratchSize are write-once on startup.